package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "openfisca.org",
+  "version": "1.0.0",
+  "description": "The public-facing website of OpenFisca for an international audience.",
+  "main": "publish.js",
```

Comment:
No! We don't want the entry point (i.e. the default script to be executed when loading this package) to deploy live!